src/mame/drivers/nes_vt.cpp

```diff
@@ -342,8 +361,13 @@ void nes_vt_state::machine_start()
 	save_item(NAME(m_timer_running));
 	save_item(NAME(m_timer_val));
 
+	m_ntram = std::make_unique<uint8_t[]>(0x2000);
+	save_pointer(NAME(m_ntram.get()), 0x2000);
+
 	m_ppu->set_scanline_callback(ppu2c0x_device::scanline_delegate(FUNC(nes_vt_state::scanline_irq),this));
 // m_ppu->set_hblank_callback(ppu2c0x_device::hblank_delegate(FUNC(device_nes_cart_interface::hblank_irq),m_cartslot->m_cart));
+//	m_ppu->space(AS_PROGRAM).install_readwrite_handler(0, 0x1fff, read8_delegate(FUNC(device_nes_cart_interface::chr_r),m_cartslot->m_cart), write8_delegate(FUNC(device_nes_cart_interface::chr_w),m_cartslot->m_cart));
+	m_ppu->space(AS_PROGRAM).install_readwrite_handler(0x2000, 0x3eff, read8_delegate(FUNC(nes_vt_state::nt_r),this), write8_delegate(FUNC(nes_vt_state::nt_w),this));
```

Comment:
because it's not plain ram.

how it gets accessed depends on registers in the system that control all sorts of mirroring and redirect.  this is here so that I have the handlers I need to add support for all of that.